CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the current maintainers of the project via email (see `Cargo.toml` file for Rust crates, `DESCRIPTION` file for R packages). All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
As the "Authorship" section of "CONTRIBUTING.md" says, authors are those who "have made multiple, sustained, and/or non-trivial contributions to the project." Just being listed on authors field doesn't mean that they have the will and bandwidth to maintain the project.

Reply:
The original document I adapted this from envisioned providing a contact email here. We can always just list @andy-thomason's email. :-) More seriously, though, I think it's always best to have multiple contact points. I'm open to suggestions on how to set this up better.

As an example, the ggplot2 code of conduct uses the following wording:
> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue, emailing the project lead (hadley@rstudio.com), or one of the core developers.

Reply:
I notice that crates.io has the concept of "owners" in addition to "authors". Maybe we say something like "see Owners on [crates.io](https://crates.io/crates/libR-sys)".